snappy_wrappers/wrappers/star/wrapper.py

```diff
@@ -137,6 +138,16 @@
         trim_cmd="zcat"
     fi
 
+    quant_mode=""
```

Comment:
What happens when someone adds `path_features` as well as `transcriptome=TRUE`

Reply:
The wrapper's `quant_mode` variable is aggregated with values `GeneCounts` & `TranscriptomeSAM` when the configuration options `path_features` & `transcriptome` resp. are set. So `quant_mode` can be empty, or take the vaules `GeneCounts`, `TranscriptomeSAM` or `GeneCounts TranscriptomeSAM`, depending on the configuration options.

When `quant_mode` is not empty, its value is passed to `STAR` after the `--quantMode` parameter.